sw/device/lib/usbdev.c

```diff
@@ -268,6 +269,10 @@ void usbdev_init(usbdev_ctx_t *ctx) {
 
   REG32(USBDEV_RXENABLE_SETUP()) = (1 << USBDEV_RXENABLE_SETUP_SETUP0);
   REG32(USBDEV_RXENABLE_OUT()) = (1 << USBDEV_RXENABLE_OUT_OUT0);
-
+  phy_config = (((pinflip) ? (1 << USBDEV_PHY_CONFIG_PINFLIP) : 0) |
```

Comment:
Consider writing `pinflip << USBDEV_PHY_CONFIG_PINFLIP` instead.

Reply:
Done. Sorry, I know it is in the standard now but when I started different systems represented TRUE with 1 or -1 so I got in the habit of not assuming.